### PR TITLE
gh-3099 Add Stream deployment experience using SCDF in CF

### PR DESCRIPTION
**Pull request by ghillert**
resolves https://github.com/spring-cloud/spring-cloud-dataflow/issues/3099

### CODE REVIEW

**Comment by ilayaperumalg** [comment]
I think we need to add some elaborate content on how users interact and verify stream deployment on CF environment (including the some of the CF deployer properties etc.,)

**Comment by sabbyanandan** [comment]
@ilayaperumalg @ghillert: Before we go elaborate on describing the CF deployer properties, let's please review what we have in the K8s section, and stick to a similar format and content length for consistency. 

We can always add/improve the content in incremental commits eventually.

File: content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md [hunk 1]
```diff
@@ -155,6 +159,20 @@ The runtime applications show information about where each application is runnin
 
 ### Cloud Foundry
 
+Before registering and deploying stream applications to Cloud Foundry using the instructions above, please ensure that you have an instance of Spring Cloud Data Flow sucessfully running on Cloud Foundry. Follow the [installation guide] for Cloud Foundry (%currentPath%/../installation/cloudfoundry/cf-cli) for reference.
+
+Once you have followed the steps of this chapter above and registered the apps as well as deployed the stream with its 3 orchestrated stream applications, you will see the sucessfully deployed applications in your Cloud Foundry Dashboard:
```
Comment (sabbyanandan) [inline content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md]: remove: "its 3 orchestrated" - seems odd in the line

Comment (sabbyanandan) [reply content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md]: perhaps in the end:

> ... in your Org/Space in Cloud Foundry:

File: content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md [hunk 2]
```diff
@@ -155,6 +159,20 @@ The runtime applications show information about where each application is runnin
...
+
+![Cloud Foundry Dashboard with the deployed Stream Application](images/SCDF-CF-dashboard.png)
+
+Of course, you have access to the runtime information of your stream applications in the Spring Cloud Data Flow dashboard as well. Simply click onto the `Runtime` button in the left navigation:
+
+![Data Flow Runtime Information](images/SCDF-CF-runtime.png)
+
+Besides verifying the runtime status of your stream, you should also verify the logging output produced by the `usage-cost-logger` Sink. In your Cloud Foundry dashboard, click onto the `Logs` tab of the `usage-cost-logger` sink application. The logging statements should look like the following:
```
Comment (sabbyanandan) [inline content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md]: Wherever we refer to it as "Cloud Foundry Dashboard", let's rephrase it to "Cloud Foundry Apps Manager"

File: content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md [hunk 3]
```diff
@@ -155,6 +159,20 @@ The runtime applications show information about where each application is runnin
...
+
+![Data Flow Runtime Information](images/SCDF-CF-dashboard-logging.png)
+
 ### Kubernetes
 
 **TODO - not finished**
```
Comment (sabbyanandan) [inline content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md]: Oh, I didn't know we haven't got to K8s yet!

**Comment by ghillert** [comment]
Rebased + merged with commit 6c9b160cdd2062bc383344f7a6308f721851712c
